### PR TITLE
Move MiniKF to distributions

### PR DESCRIPTION
**Pull request by RFMVasconcelos**
See #2565

### CODE REVIEW

**Comment by RFMVasconcelos** [comment]
cc @cvenets @jbottum

**Comment by RFMVasconcelos** [comment]
@8bitmp3 could you please review? This is the last distribution to move into the folder, so we'll be set :)

File: content/en/docs/distributions/minikf/minikf-aws.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 +++
-title = "Deploy Kubeflow using MiniKF on AWS"
+title = "MiniKF on AWS Marketplace"
 description = "Deploy Kubeflow with MiniKF (mini Kubeflow) via AWS"
```
Comment (cvenets) [inline content/en/docs/distributions/minikf/minikf-aws.md:3]: I would remove the parenthesis altogether here and adding "Marketplace":

"Deploy Kubeflow with MiniFK via AWS Marketplace"

File: content/en/docs/distributions/minikf/minikf-gcp.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 +++
-title = "Deploy Kubeflow using MiniKF on Google Cloud"
+title = "MiniKF on GCP Marketplace"
 description = "Deploy Kubeflow with MiniKF (mini Kubeflow) via Google Cloud Marketplace"
```
Comment (cvenets) [inline content/en/docs/distributions/minikf/minikf-gcp.md:3]: I would remove the parenthesis altogether here:

"Deploy Kubeflow with MiniFK via GCP Marketplace"

so it is unified with the above.

File: content/en/docs/distributions/minikf/minikf-aws.md (hunk 1 above)
Comment (RFMVasconcelos) [reply content/en/docs/distributions/minikf/minikf-aws.md:3]: @cvenets On the description I kept it as is today, but happy to make this change :)

**Comment by RFMVasconcelos** [comment]
@cvenets I've updated this according to your suggestions, can you please review and LGTM?

**Comment by cvenets** [comment]
@RFMVasconcelos thank you. It looks good to me.

**Pull request merged**